leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -714,17 +744,41 @@ private boolean checkPlayServices() {
     return true;
   }
 
+  private void setLocationInfo(double lng, double lat) {
+    this.longitude.setText(getString(R.string.longitude) + ": " + lng);
```

Comment:
I think that all the small mini-methods also don't assist on readability.
You now have:
- changeLocationInfoLayoutVisibility
- updateAddressLayoutVisibility
- changeListResultVisibility
- willGetLocationInfo
- showLastLocation
- showLocationInfo
- showLocationInfoLayout
- setLocationInfo

It's my opinion, but I think it gets more and more spageti this way